pom.xml

```
@@ -1620,6 +1620,104 @@
                 </plugins>
             </build>
         </profile>
+		<profile>
+			<id>java8</id>
+			<activation>
+				<jdk>1.8</jdk>
+			</activation>
+			<build>
+				<plugins>
+					<plugin>
+						<groupId>org.apache.maven.plugins</groupId>
+						<artifactId>maven-enforcer-plugin</artifactId>
+						<version>3.0.0-M3</version>
+						<executions>
+							<execution>
+								<id>enforce-versions</id>
+								<phase>validate</phase>
+								<goals>
+									<goal>enforce</goal>
+								</goals>
+								<configuration>
+									<rules>
+										<requireJavaVersion>
+											<version>[1.8.0-241,)</version>
+											<message>Strongbox only supports JDK 11.3 and up.</</message>
+											<level>ERROR</level>
+										</requireJavaVersion>
```

Please, change this to:

```suggestion
											<version>[1.8.0-241,)</version>
											<message>Strongbox only supports JDK 1.8.0-241, or higher.</message>
											<level>ERROR</level>
```